spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe 'User Stories' do
+
+  let(:menu)  { Menu.new  }
+  let(:order) { Order.new(menu.options) }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'customers can see a list of items with prices' do
+    expect(menu.options).to eq({"beef-chuck brisket sub"=>4.5, "pastrami salami sub"=>5.0})
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.